src/index.js

```diff
@@ -8,38 +8,18 @@ const reloadIcon = document.getElementById('reload-img');
 const enterIcon = document.getElementById('enter-icon');
 enterIcon.src = enter;
 reloadIcon.src = reload;
-const todoList = [
-  {
-    id: 1,
-    description: 'Starting project structure',
-    completed: false,
-  },
-  {
-    id: 2,
-    description: 'Initiate writing JS code',
-    completed: false,
-  },
-  {
-    id: 3,
-    description: 'Wash my Dog',
-    completed: true,
-  },
-  {
-    id: 4,
-    description: 'Pray ',
-    completed: false,
-  },
-];
+
+let todoList = [];
 
 const renderList = (list) => {
   let innerList = '';
   if (list.length === 0) {
-    innerList = '<h3 class="list-placeholder">Add your first task!</h3>';
+    innerList = '<h3 class="list-placeholder">Please, add your first task!</h3>';
   } else {
-    const sortedList = list.sort((a, b) => a.id - b.id);
+    const sortedList = list.sort((a, b) => a.index - b.index);
     sortedList.forEach((task) => {
       innerList += `
-          <li class="to-do-tasks">
+          <li class="to-do-tasks" data-task-id="${task.id}">
             <div class="check-box ${task.completed ? 'completed' : ''}">${task.completed ? '✓' : ''}</div>
             <p class="task-description ${task.completed ? 'line-through' : ''}">${task.description}</p>
             <img class="more-logo" src="${moreIcon}"/>
```

Comment:
This icon, when clicked should at least show an icon that makes deleting a single todo possible, I recommend including this as it is a requirement

Reply:
Dit it!!!, thanks!!!

![image](https://github.com/lerfast/To-do-list-webpack/assets/111823606/6a902440-6a34-4772-a0e7-a5e9fa56c458)